README.md

```diff
@@ -85,8 +71,48 @@ IS LAST PAGE				true
 
 ## Relations
 
+### PostgreSQL
+
 This charm requires a relation with [postgresql-k8s-operator](https://github.com/canonical/postgresql-k8s-operator).
 
+### Ingress
+
+The Hydra Operator offers integration with the [traefik-k8s-operator](https://github.com/canonical/traefik-k8s-operator) for ingress.
+Hydra has two APIs which can be exposed through ingress, the public API and the admin API.
+
+If you have traefik deployed and configured in your hydra model, to provide ingress to the admin API run:
+```console
```

Comment:
This somehow suggests that we need 2 traefik operators deployed. Is this accurate?

Reply:
Just to be clear, we don't _need_ it but the current traefik/ingress relation interface does not allow us to have expose 2 ports exposed from the same traefik instance. 

Also you don't need to expose the admin API to have a working hydra installation.

Reply:
Hmm. This is a bit odd - @PietroPasotti do you have any insight here?

Feels like we could deploy with 1x `traefik-k8s` and 2x `traefik-route-k8s` (one for each endpoint)?

Reply:
Ummm I think you should be able to configure a single traefik-route instance to expose multiple ports, or indeed, you'd need two traefik instances, one for each port/endpoint.

Reply:
I could be wrong but I believe that the initial decision of using 2 x traefik-route-k8s was taken after a discussion about the traefik-k8s limitations with Dominik and @simskij 
Happy to update our approach if you can point us to a working implementation or provide more guidance